site.js

```diff
@@ -1512,28 +1652,39 @@ function generateCurse() {
 		},
 		{
 			subjectText: "naga",
+			chosen: function(){extemitiesName = "claws"; facialfeatureName = "crest";},
+			additionalExplaination: randomFrom([
+					"Holding a lover or helpless victim in your coils is such a delight.",
+					"",
+					]),
+			closingRemarkText: "I hope you don't mind having a lisssp.",
 			sets: [subjectInhuman, setPussyName("scaled ".concat(pussyName))],
 			requires: [genderAgnostic, notBecomingHybrid, nonMundaneSubject, humanoidOption],
 		},
 		{
-			subjectText: "cerberus",
+			subjectText: randomFrom(["cerberus", "hydra"]),
+			chosen: function(){facialfeatureName = "horns";},

```

Comment:
cerberuses don't have horns. . . .. I guess we can let this slide since they're sometimes depicted that way.